meta/main.yml

```diff
@@ -13,5 +14,6 @@ galaxy_info:
       versions:
         - precise
         - trusty
+        - xenial
```

Comment:
As Xenial is supported now. I would add the same `.make` file documentation into the doc. https://github.com/sansible/kong/blob/3f16a65f6aa617306c0123c1059003f50bb22d62/README.md